widget_bandsplot/schemas/pdos.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Is this file referenced anywhere in the code? You could use python-jsonschema to validate input against this schema.

Reply:
Yes, I use it in the test and for the validation of the example JSON file by pre-commit.

Reply:
What about my suggestion to validate the input during runtime?

Reply:
I see, good point! I add the check when load the dos inputs in `BandsWidget` Class, please have a look at the latest commit https://github.com/osscar-org/widget-bandsplot/pull/18/commits/0104b8cb44e585a2eaedd32a288a3c99c730be15

Reply:
Great, thanks!